hana/extract_hdbserver.sls

```diff
@@ -0,0 +1,22 @@
+{%- from "hana/map.jinja" import hana with context -%}
+{% set host = grains['host'] %}
```

Comment:
We don't need this `host` variable, right?
If we can, I would remove it to make everything simpler